modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/source/ExtensionMessageSource.java

```diff
@@ -747,4 +707,12 @@ private void stopUsingConfiguration(CoreEvent event) {
   public String toString() {
     return this.getClass().getSimpleName() + ": " + Objects.toString(sourceAdapter);
   }
+
+  private class ImmediateExecutor implements Executor {
```

Comment:
delete and use org.mule.runtime.core.internal.util.rx.ImmediateScheduler#IMMEDIATE_SCHEDULER instead